tests/python/test_uds.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+from parameterized import parameterized_class
+import unittest
+import struct
+import threading
+
+from panda.python.uds import SERVICE_TYPE, DATA_IDENTIFIER_TYPE, IsoTpMessage, UdsClient, get_rx_addr_for_tx_addr
+
+DEFAULT_VIN = '1H3110W0RLD5'
+
+
+class UdsServer(UdsClient):
```

Comment:
I would probably separate this class from the unit tests, it is useful for other things like building a fake car for playing around with the dealer tools.